### PR TITLE
Fix service name in dev setup

### PR DESCRIPTION
**Pull request by tillprochaska**

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -23,14 +23,13 @@ services:
...
-    depends_on: [ db, mailserver ]
+    depends_on: [ db, signal, mailserver ]
     ports:
       - 3000:3000
       - 3035:3035
 
-  queue:
+  job-worker:
     <<: *x-dev-defaults
-    depends_on: [ db ]
```
Comment (roschaefer) [inline docker-compose.override.yml:33]: The job-worker does not depend on the `db`? :thinking: Am I missing something? https://github.com/collectiveidea/delayed_job#installation

**Review by roschaefer — CHANGES_REQUESTED** [review]
Sorry @tillprochaska here's another request changes.

File: docker-compose.override.yml (hunk 1 above)
Comment (tillprochaska) [reply docker-compose.override.yml:33]: It does actually depend on the `db` and `signal` services. That’s already defined in `docker-compose.yml` though (see [this line](https://github.com/tactilenews/100eyes/blob/master/docker-compose.yml#L21)), so there’s no need to overwrite it in `docker-compose.override.yml`.

Comment (tillprochaska) [reply docker-compose.override.yml:33]: The change in the dependencies for the `app` services wasn’t necessary though, as the `app` service never interacts with the `signal` service directly, it only enqueues jobs that then send requests to the `signal` service.

**Review by roschaefer — APPROVED** [review]
LGTM! :+1:

**Pull request merged**